flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/RedisSinkITCase.java

```diff
@@ -127,6 +146,24 @@ public void testRedisHashDataType() throws Exception {
         env.execute("Test Redis Hash Data Type");
 
         assertEquals(NUM_ELEMENTS, jedis.hlen(REDIS_ADDITIONAL_KEY));
+        Thread.sleep(REDIS_TTL * 1000);
+        assertEquals(true, jedis.exists(REDIS_ADDITIONAL_KEY));
```

Comment:
You can use `jedis.ttl` to check that a key has an associated ttl instead of sleeping